src/main/java/org/broadinstitute/hellbender/engine/filters/MappingQualityReadFilter.java

```diff
@@ -14,18 +14,28 @@
 public final class MappingQualityReadFilter extends ReadFilter implements Serializable {
     private static final long serialVersionUID = 1L;
 
-    @Argument(fullName="minimumMappingQuality", shortName="minimumMappingQuality", optional=true)
+    @Argument(fullName="minimumMappingQuality", shortName="minimumMappingQuality", doc = "Minimum mapping quality to keep (inclusive)", optional=true)
     public int minMappingQualityScore = 10;
 
+    @Argument(fullName="maximumMappingQuality", shortName="maximumMappingQuality", doc = "Maximum mapping quality to keep (exclusive)", optional=true)
```

Comment:
Sorry, I think that I changed the implementation because it was confusing and I forgot to update the text.